components/math/multiplicationConnect/GameBoardBlock.tsx

```diff
@@ -1,20 +1,20 @@
 import React, {useState} from 'react'
+import { calculateWinner } from '../../../pages/studentPortal/labs/multiplication-connect/Index';
 
 
-const blockClick = (block) => {
-    (block.isSelected===false) ? block.isSelected=true : block.isSelected=false;
-    // console.log(block);
+const blockClick = (block, grid) => {
+    (block.isSelected===false) ? (block.isSelected=true, calculateWinner(grid)) : block.isSelected=false;
```

Comment:
I would say use a normal if/else statement if your clauses are more than just one line. Since you have a clause that's doing two things (ie. setting isSelected to true and you're running calculate winner after that, I would re-write this to use a regular if/else statement.

Reply:
Sounds good